server-mock/server/get-whitelist.go

```diff
@@ -1,15 +1,45 @@
 package server
 
 import (
+	"fmt"
 	"net/http"
 
+	"github.com/ethereum/go-ethereum/crypto"
 	apiv2 "github.com/nodeset-org/nodeset-client-go/api-v2"
+	"github.com/nodeset-org/nodeset-client-go/server-mock/db"
+	nsutil "github.com/nodeset-org/nodeset-client-go/utils"
+	"github.com/rocket-pool/node-manager-core/utils"
 )
 
 func (s *NodeSetMockServer) getWhitelist(w http.ResponseWriter, r *http.Request) {
 	data := apiv2.WhitelistData{}
+
+	// Get the requesting node
+	session := s.processAuthHeader(w, r)
+	if session == nil {
+		return
+	}
+	node := s.getNodeForSession(w, session)
+	if node == nil {
+		return
+	}
+
+	db := db.NewDatabase(s.logger)
+	privateKey, err := crypto.ToECDSA(db.ConstellationAdminPrivateKey)
```

Comment:
Yeah if the key is already stored in this format you don't have to convert it with every call.

Reply:
updated